cardano-config/src/Cardano/Config/Shelley/Orphans.hs

```diff
@@ -218,6 +218,12 @@ deriving newtype instance Crypto c => ToJSON (UTxO c)
 deriving newtype instance ToJSON   Coin
 deriving newtype instance FromJSON Coin
 
+deriving newtype instance ToJSON   SlotNo
+deriving newtype instance FromJSON SlotNo
+
+deriving newtype instance ToJSON   EpochNo
+deriving newtype instance FromJSON EpochNo
+
```

Comment:
Why did these get moved from `Common` to `Shelley`? They are common types, not Shelley specific.